### PR TITLE
Update Cache_memcached.php typo error

### PR DESCRIPTION
**Pull request by cweric**

### CODE REVIEW

File: system/libraries/Cache/drivers/Cache_memcached.php [hunk 1]
```diff
@@ -295,4 +295,4 @@ public function is_supported()
 }
 
 /* End of file Cache_memcached.php */
-/* Location: ./system/libraries/Cache/drivers/Cache_memcached.php */
\ No newline at end of file
+/* Location: ./system/libraries/Cache/drivers/Cache_memcached.php */
```
Comment (narfbg) [inline system/libraries/Cache/drivers/Cache_memcached.php]: Please remove the empty line here (you have to use a local editor, can't happen through GitHub's).

**Comment by ZombieProtectionAgency** [comment]
@narfbg Sorry if i shouldn't ask this here, but I have been wondering since yesterday and since it came up....
>you have to use a local editor, can't happen through GitHub's 
 
How exactly do you do that? Is there any way to edit a single file with a local editor or do you have to install git and everything locally first?

**Comment by vlakoff** [comment]
This issue should have been kept open, @narfbg you might want to directly fix the bug. It's the most straightforward.

**Comment by narfbg** [comment]
No shit. :)

**Pull request closed without merging**